src/common/dns_resolve.cc

```diff
@@ -314,7 +314,7 @@ int DNSResolver::resolve_srv_hosts(CephContext *cct, const string& service_name,
   }
 #endif
   if (len < 0) {
-    lderr(cct) << "res_search() failed" << dendl;
+    lderr(cct) << "res_search() failed" << query_str << dendl;
```

Comment:
@joscollin thanks for you advise
but I think I only focus on the contents of “query_str”, and can be consistent with the subsequent print, so I tend to modify the current way.